test/adapter/json_api/transform_test.rb

```diff
@@ -102,8 +102,8 @@ def test_success_document_transform_global_config
           mock_request
           result = with_config(key_transform: :camel_lower) do
             serializer = PostSerializer.new(@post)
-            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer)
-            adapter.serializable_hash(@options)
+            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer, @options)
```

Comment:
and, omg, I can't read, that's where it's been all along.  no test pollution, just `test/adapter/json_api/transform_test.rb` looks a lot like `test/action_controller/json_api/transform_test.rb`